test/spec/windshaft/map-serializer/anonymous-map-serializer/layers-serializer.spec.js

```diff
@@ -0,0 +1,116 @@
+var Backbone = require('backbone');
+var CartoDBLayer = require('../../../../../src/geo/map/cartodb-layer');
+var LayersSerializer = require('../../../../../src/windshaft/map-serializer/anonymous-map-serializer/layers-serializer');
+var PlainLayer = require('../../../../../src/geo/map/plain-layer');
+var TileLayer = require('../../../../../src/geo/map/tile-layer');
+var TorqueLayer = require('../../../../../src/geo/map/torque-layer');
+var VisModel = require('../../../../../src/vis/vis');
+
+describe('layers-serializer', function () {
+  describe('.serialize', function () {
+    var visMock;
```

Comment:
We tend to use `this.foo` inside the `beforeEach` instead of using variables. For example:
https://github.com/CartoDB/cartodb.js/pull/1779/files#diff-6c3f2d03eeffeb930a0da94cbd134a5fR11
I don't really prefer one over the other (perhaps using vars makes our lives simpler...), but we should keep it consistent.

@IagoLast It seem you prefer `var` over `this.`, right? Why?

Reply:
when using `this.foo` the linter cannot determinate if the variable is declared or unused plus, using `this`we must handle the context manually (ie: inside a loop) so IMO using var is less error prone.